trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -2,14 +2,19 @@ import { AMENDMENT_STATUS, isTfmFacilityEndDateFeatureFlagEnabled } from '@ukef/
 import caseController from '.';
 import api from '../../api';
 import { mockRes } from '../../test-mocks';
-import { getTask } from '../helpers';
+import { dealCancellationEnabled, getTask } from '../helpers';
 import mapAssignToSelectOptions from '../../helpers/map-assign-to-select-options';
 
 jest.mock('@ukef/dtfs2-common', () => ({
   ...jest.requireActual('@ukef/dtfs2-common'),
   isTfmFacilityEndDateFeatureFlagEnabled: jest.fn(),
 }));
 
+jest.mock('../helpers', () => ({
```

Comment:
Is this `env` variable mock, not sure if this is required anymore?

Reply:
This is mocking the return result of `dealCancellationEnabled` - given this is a unit test of the `index` file which doesn't handle the `env` values, I'm just mocking the return result of the function it calls, and the different feature flag values are instead unit tested within the `dealCancellationEnabled` test files.